crates/apub/src/activities/create_or_update/comment.rs

```diff
@@ -178,8 +179,10 @@ impl ActivityHandler for CreateOrUpdateNote {
     // anyway.
     // TODO: for compatibility with other projects, it would be much better to read this from cc or
     // tags
-    let mentions = scrape_text_for_mentions(&comment.content);
-    send_local_notifs(mentions, comment.id, &actor, do_send_email, context).await?;
+    if let Some(local_user) = LocalUserView::read_person(&mut context.pool(), actor.id).await? {
```

Comment:
This check probably should've been done already, but now it makes sure that the local user exists before sending notifs.